examples/networking/vlan_filter/data-plane-tracing.py

```diff
@@ -0,0 +1,193 @@
+
+from sys import argv
+from kafka import KafkaProducer
+from kafka.errors import KafkaError
+from datetime import datetime
+
+#args
+def usage():
+    print("USAGE: %s [-i <if_name>]" % argv[0])
+    print("")
+    print("Try '%s -h' for more options." % argv[0])
+    exit()
+
+#help
+def help():
+    print("USAGE: %s [-i <if_name>][-k <kafka_server_name:kafka_port>]" % argv[0])
+    print("")
+    print("optional arguments:")
+    print("   -h                       print this help")
+    print("   -i if_name               select interface if_name. Default is eth0")
+    print("   -k kafka_server_name     select kafka server name. Default is save to file")
+    print("                            If -k option is not specified data will be saved to file.")
+    
+    print("")
+    print("examples:")
+    print("    data-plane-tracing                                      # bind socket to eth0")
+    print("    data-plane-tracing -i eno2 -k vc.manage.overcloud:9092  # bind socket to eno2 and send data to kafka server in iovisor-topic.")
+    exit()
+
+#arguments
+interface="eth0"
+kafkaserver=''
+        
+#check provided arguments
+if len(argv) == 2:
+    if str(argv[1]) == '-h':
+        help()
+    else:
+        usage()
+
+if len(argv) == 3:
+    if str(argv[1]) == '-i':
+        interface = argv[2]
+    elif str(argv[1]) == '-k':
+        kafkaserver = argv[2] 
+    else:
+        usage()
+    
+if len(argv) == 5:
+    if str(argv[1]) == '-i':
+        interface = argv[2]
+        kafkaserver = argv[4]
+    elif str(argv[1]) == '-k':
+        kafkaserver = argv[2] 
+        interface = argv[4]
+    else:
+        usage()
+
+if len(argv) > 5:
+    usage()
+
+print ("binding socket to '%s'" % interface)	
+ 
+#initialize BPF - load source code from http-parse-simple.c
+bpf = BPF(src_file = "data-plane-tracing.c", debug = 0)
+
+#load eBPF program http_filter of type SOCKET_FILTER into the kernel eBPF vm
+#more info about eBPF program types http://man7.org/linux/man-pages/man2/bpf.2.html
+function_vlan_filter = bpf.load_func("vlan_filter", BPF.SOCKET_FILTER)
+
+#create raw socket, bind it to eth0
+#attach bpf program to socket created
+BPF.attach_raw_socket(function_vlan_filter, interface)
+
+#get file descriptor of the socket previously created inside BPF.attach_raw_socket
+socket_fd = function_vlan_filter.sock
+
+#create python socket object, from the file descriptor
+sock = socket.fromfd(socket_fd,socket.PF_PACKET,socket.SOCK_RAW,socket.IPPROTO_IP)
+
+#set it as blocking socket
+sock.setblocking(True)
+
+#get interface ip address. In case ip is not set then just add 127.0.0.1.
+ni.ifaddresses(interface)
+try:
+    ip = ni.ifaddresses(interface)[ni.AF_INET][0]['addr']
+except:
+    ip = '127.0.0.1'    
+
+print("timestamp, hosname, MachineIP   ipver     Src-Host IP Addr     Dst-Host IP Addr     Src-Host_Port   Des-Host_Port  Src-VM_Addr   Dest-VM_Addr     Src-VM_Port   Dest-VM_Port VNI  VLANID  protocol  Packet Length ")
+
```

Comment:
The above header printout does not match the actual output and README. Could you fix it? Otherwise, the patch looks good. Thanks.

Reply:
I have made requested update to reflect same header printout with actual output and README. 
Also, I would like to thank you for all your guidance and suggestions throughout this process.